docs/_static/didman/v1.yaml

```diff
@@ -213,6 +213,37 @@ paths:
           description: The service has been removed
         default:
           $ref: '../common/error_response.yaml'
+  /internal/didman/v1/search/organizations:
+    parameters:
+      - name: query
+        in: query
+        description: Query used for searching the organization by name.
```

Comment:
Can this be a subset of the organization name? like `oodvin`
Is it case sensitive?`zorg`

Reply:
Yes, I'll further document its behavior